src/towncrier/newsfragments/608.feature.rst

```diff
@@ -0,0 +1,8 @@
+Issues are now sorted by issue number even if they have non-digit characters.
```

Comment:
```suggestion
News fragments are now sorted by ticket number even if they have non-digit characters.
```

Maybe we should update the [Philosophy](https://towncrier.readthedocs.io/en/latest/#philosophy) paragram in the documentation to also introduce the "issue number"

In the past we had Trac `tickets`, now we have GitHub `issues` ... and I think some people have Jira `tickets`


I see that in [towncrier create](https://towncrier.readthedocs.io/en/latest/cli.html#towncrier-create) doc we have 

> To create a news fragment not tied to a specific issue

----------

But in [the tutorial](https://towncrier.readthedocs.io/en/latest/tutorial.html#creating-news-fragments) I see 

> misc: A ticket has been closed, but it is not of interest to users.

> When you create a news fragment, the filename consists of the ticket ID (or some other unique identifier) as well as the ‘type’. 

> For orphan news fragments (those that don’t need to be linked to any ticket ID or other identifier), 


---------------

I think that it's important to have consistent terminology.

For the purpose of being generic, I would prefer to use the term `ticket`